src/maker/rpc/server.rs

```diff
@@ -111,6 +115,34 @@ fn handle_request(maker: &Arc<Maker>, socket: &mut TcpStream) -> Result<(), Make
                 log::info!("Error sending RPC response {:?}", e);
             };
         }
+        RpcMsgReq::SendToAddress {
+            address,
+            amount,
+            fee,
+        } => {
+            let amount = Amount::from_sat(amount);
+            let fee = Amount::from_sat(fee);
+            let destination =
+                Destination::Address(Address::from_str(&address).unwrap().assume_checked());
+
+            let coins_to_send = maker.get_wallet().read()?.coin_select(amount + fee)?;
+
+            let tx = maker.get_wallet().write()?.spend_from_wallet(
+                fee,
+                SendAmount::Amount(amount),
+                destination,
+                &coins_to_send,
+            )?;
+
+            let calculated_fee_rate = fee / (tx.weight());
+            println!("Calculated FeeRate : {:#}", calculated_fee_rate);
```

Comment:
Use logger for these... rest looks ok. Did it worked for you locally?

Reply:
not yet, makerd seems broken to me right now, waiting for #291 to get merged, should work after that